webpack.config.js

```diff
@@ -6,14 +6,19 @@ const webpack = require('webpack');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const nodeExternals = require('webpack-node-externals');
 
+// ESM packages must be bundled because webpack will otherwise turn `require()` into `import()`
+// and make the import silently asynchronous. In most cases this breaks the import.
+const dependenciesToBundle = [
+  'os-locale', // Used exclusively in sync functions
+];
```

Comment:
As expected, there's "no way" to `require` a ESM package synchronously. This is not a webpack/config issue. The only way for this to work is to turn `addons-linter` into an `type=module` package.

So this is the alternative: _do not_ keep `os-locale` as an external dependency, so that node does not try to `require` it at runtime.